src/main/java/roomescape/service/MemberService.java

```diff
@@ -39,10 +41,10 @@ private void checkDuplicateEmail(String email) {
     }
 
     @Transactional
-    public void withdrawal(Long memberId) {
+    public void deleteMember(Long memberId) {
         Member findMember = memberRepository.findById(memberId)
                 .orElseThrow(IllegalArgumentException::new);
-
+        waitRepository.deleteByMemberId(memberId);
```

Comment:
```suggestion
        waitRepository.deleteByMember(findMember);
```
spring-data-jpa를 사용한다면 이렇게 넣어도 가능합니다

Reply:
[리뷰 반영 완료] 클라이언트에서 entity Id를 바로 받아오는 경우를 제외하면 모두 코멘트와 같이 수정하였습니다.
(즉, select query로 조회 조건의 인스턴스를 가져와야하는 경우 제외)